server/db/FirestoreDB.go

```diff
@@ -53,15 +53,26 @@ func (db *firestoreDB) HasGameByID(id string) bool {
 }
 
 // CreateGame a game with the given ID. Perhaps this should instead just return an id?
-func (db *firestoreDB) CreateGame() (*model.Game, error) {
-	game := model.Game{ID: uuid.New().String(), Password: "12234"}
-	gameDoc := db.games.Doc(game.ID)
+func (db *firestoreDB) CreateGame(gameName string, creatorID string) (*model.Game, error) {
+	player, err := db.LookupPlayer(creatorID)
+	if err != nil {
+		return nil, err
+	}
+
+	myGame := model.Game{
+		ID:       uuid.New().String(),
+		Password: "12234",
+		Creator:  *player,
```

Comment:
do you want the whole *player here? or just the id?

Reply:
As I read the model, the whole player object is required. _Should_ it be required is a different question...

Reply:
I'm fine with that response :P